packages/core-storage/lib/storage.js

```diff
@@ -1,24 +1,47 @@
 'use strict'
 
 const immutable = require('immutable')
-const { get, set, has, omit } = require('lodash')
 
 module.exports = class Storage {
   /**
    * Create a new Storage instance.
    */
   constructor () {
-    this.storage = {}
+    this.storage = immutable.Map()
   }
 
   /**
-   * Get all of the storage data.
-   * @return {[type]}
+   * Get the storage map.
+   * @return {Object}
    */
   all () {
     return this.storage
   }
 
+  /**
```

Comment:
Perhaps these might be better as get methods?:

Reply:
It is a method to mimic the behaviour of the underlying `immutable.js` library as I want to keep the interface the same because the storage is only a wrapper.

Reply:
Ah okay, if it's to match that then it makes sense 👌